tests/schemas/root.json

```diff
@@ -2,10 +2,19 @@
     "type": "object",
     "properties": {
         "readme": {"type": "string"},
-        "features": {"type": "object"},
+        "features": {
+            "type": "object",
+            "patternProperties": {
+                "^[a-zA-Z0-9]+$": {"type": "object"}
```

Comment:
this makes sure that feature name is using only [a-zA-Z0-9] in its name. We could paste whole `feature.json` in here then, but I'm OK with keeping it separate